$NetBSD: patch-ah,v 1.3 2011/10/09 06:40:38 dholland Exp $

- use pkg-config to find libXft, not xft-config, as xft-config has
disappeared from recent libXft. (XXX: will this break the build for
really old libXft? Does this need to learn to use both?)

- use proper pthreads. (warning: this change is not reflected in
the configure.in patch)

--- configure.orig	2005-04-29 16:31:42.000000000 +0000
+++ configure
@@ -5513,8 +5513,8 @@ if test "${enable_xft+set}" = set; then
 #define USE_XFT 1
 _ACEOF
 
-	LDLIBS="$LDLIBS `xft-config --libs`"
-        CXXFLAGS="$CXXFLAGS `xft-config --cflags`"
+	LDLIBS="$LDLIBS `pkg-config --libs xft`"
+        CXXFLAGS="$CXXFLAGS `pkg-config --cflags xft`"
 fi
 fi;
 
@@ -8295,13 +8295,13 @@ if test $ac_cv_lib_c_pthread_create = ye
   \
      have_pthread=yes
 else
-  echo "$as_me:$LINENO: checking for pthread_create in -lpthread" >&5
-echo $ECHO_N "checking for pthread_create in -lpthread... $ECHO_C" >&6
+  echo "$as_me:$LINENO: checking for pthread_create in ${PTHREAD_LDFLAGS} ${PTHREAD_LIBS}" >&5
+echo $ECHO_N "checking for pthread_create in ${PTHREAD_LDFLAGS} ${PTHREAD_LIBS}... $ECHO_C" >&6
 if test "${ac_cv_lib_pthread_pthread_create+set}" = set; then
   echo $ECHO_N "(cached) $ECHO_C" >&6
 else
   ac_check_lib_save_LIBS=$LIBS
-LIBS="-lpthread  $LIBS"
+LIBS="${PTHREAD_LDFLAGS} ${PTHREAD_LIBS}  $LIBS"
 cat >conftest.$ac_ext <<_ACEOF
 /* confdefs.h.  */
 _ACEOF
@@ -8361,7 +8361,7 @@ echo "$as_me:$LINENO: result: $ac_cv_lib
 echo "${ECHO_T}$ac_cv_lib_pthread_pthread_create" >&6
 if test $ac_cv_lib_pthread_pthread_create = yes; then
   \
-        THREADSLIBS=-lpthread have_pthread=yes
+        THREADSLIBS="${PTHREAD_LDFLAGS} ${PTHREAD_LIBS}" have_pthread=yes
 fi
 
 fi
